test/runtime/specs/juttle-spec/expressions/function-call.spec.md

```diff
@@ -151,3 +151,106 @@ Produces an error when passed too many parameters (module)
 ### Errors
 
   * Error: function m.f expects 1 to 2 arguments but was called with 3
+
+Returns correct result when passed required parameters (without optional parameters)
+---------------------------------------------------------------------------------
```

Comment:
Make `-`’s match the length of the title?

(We should probably switch to `##` for JuttleSpec headers globally at some point, managing the `-`’s is pain.)